### DISCUSSION TITLE
Have raw tf plan output

### DISCUSSION THREAD

**Discussion by ibrt2018** [dflook/terraform-github-actions#346] [General]
Hello,

Is it possible to have an output of raw tf plan file exactly like terraform plan  -input=false -no-color -out=tfplan?
If so, how can I add it in the action?

**Comment by dflook** [comment]
Yes, see the [plan_path](https://github.com/dflook/terraform-github-actions/tree/main/terraform-plan#outputs) output of the plan action. See also the `json_plan_path` and `text_plan_path` outputs for other formats.

**Reply by ibrt2018** [reply]
alright thank you very much @dflook
